genesyscloud/resource_genesyscloud_flow.go

```diff
@@ -92,6 +92,16 @@ func ResourceFlow() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"flow_name": {
+				Description: `Genesys Cloud flow name`,
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
+			"flow_type": {
+				Description: `Genesys Cloud flow type`,
+				Type:        schema.TypeString,
+				Optional:    true,
+			},
```

Comment:
I would mark this as Computed. If the user provides no value and one is read into the schema in the read function, they should see an error.